data/data/baremetal/masters/main.tf

```diff
@@ -3,6 +3,7 @@ resource "ironic_node_v1" "openshift-master-host" {
   name           = var.hosts[count.index]["name"]
   resource_class = "baremetal"
 
+  inspect   = true
```

Comment:
You can probably skip all of the vendoring and terraform changes, except for setting "inspect = true".

We don't need terraform to know about inspector URI, or have a data source. We're going to gather the data directly from ironic on the bootstrap. That should make this PR simpler.

Reply:
OK will do.  Thanks